pkg/knuu/knuu.go

```diff
@@ -49,8 +49,6 @@ type Options struct {
 }
 
 func New(ctx context.Context, opts Options) (*Knuu, error) {
-	opts.TestScope = k8s.SanitizeName(opts.TestScope)
```

Comment:
This was supposed to be removed in an already merged PR, but it is still there an causing panics. So, it is removed even though it is not relevant to this PR.

Reply:
If this causes panic because `SanitizeName` fails if `opts.TestScope` is an empty string, we could add a check for that case.

Reply:
The decision was to set a default scope when it is empty and it is checked in the `setDefaults` func. That's why we have it removed from the beginning of the `New` func.